src/Microsoft.ComponentDetection.Detectors/pnpm/PnpmComponentDetector.cs

```diff
@@ -30,7 +31,7 @@ public class PnpmComponentDetector : FileComponentDetector
 
     public override IEnumerable<ComponentType> SupportedComponentTypes { get; } = new[] { ComponentType.Npm };
 
-    public override int Version { get; } = 5;
+    public override int Version { get; } = 6;
```

Comment:
I think I'm supposed to update this because I made changes, but I'd like confirmation that that's the correct thing to do.

Reply:
I'd be happy to help make these changes as well if needed

Reply:
That sounds like an excellent way to reduce risk in cases like this.

> I'd be happy to help make these changes as well if needed

That would be great. I don't know this codebase well (I just spent a day and a half making this as part of a hackathon), so I suspect you could do that much more efficiently than I could. I'm happy to have you take over any part or all of this work.